lib/useDeferredList/useDeferredList.js

```diff
@@ -8,10 +8,6 @@ function useDeferredList({
   const [itemCount, setItemCount] = useState(initialItemCount);
 
   const setItemCountWithinLimits = count => {
-    if (count === listLength) {
```

Comment:
This was stopping us updating our list to the final count value!

Reply:
Why two rerenders meant that it did update, I don't know! But we don't need it anymore!